UnboxTests.swift

```diff
@@ -865,6 +906,134 @@ private struct UnboxTestSimpleMock: Unboxable, Equatable {
+      "labore",
+      "sit",
+      "nulla",
+      "Lorem"
+    ],
+    "meta": {
+      "link": {
+        "self": "http://example.com/amet",
+        "author": "http://example.com/sunt",
+        "collection": "http://example.com/reprehenderit",
+        "replies": "http://example.com/commodo"
+      }
+    }
+  },
+  {
+    "id": "56fb503151f18f3ebb06b7e3",
+    "name": "Bruce Keith",
+    "isActive": false,
+    "age": 25,
+    "registered": "2015-07-22T11:05:09",
+    "tags": [
+      "ea",
+      "tempor",
+      "laboris",
+      "nostrud",
+      "magna",
+      "aute",
+      "magna"
+    ],
+    "meta": {
+      "link": {
+        "self": "http://example.com/non",
+        "author": "http://example.com/eiusmod",
+        "collection": "http://example.com/ad",
+        "replies": "http://example.com/eiusmod"
+      }
+    }
+  }
+]
+*/
+// Minified above then escaped quotes: http://www.httputility.net/json-minifier.aspx
+private let JSONMockString: String = "[{\"id\":\"56fb5031affd22691ea32b92\",\"name\":\"Clare Lancaster\",\"isActive\":true,\"age\":27,\"registered\":\"2014-07-14T07:04:50\",\"tags\":[\"magna\",\"velit\",\"qui\",\"ea\",\"fugiat\",\"est\",\"eiusmod\"],\"meta\":{\"link\":{\"self\":\"http://example.com/ex\",\"author\":\"http://example.com/exercitation\",\"collection\":\"http://example.com/enim\",\"replies\":\"http://example.com/fugiat\"}}},{\"id\":\"56fb50316dacc51823381a99\",\"name\":\"Angela Chan\",\"isActive\":true,\"age\":22,\"registered\":\"2016-02-09T04:09:05\",\"tags\":[\"sunt\",\"et\",\"commodo\",\"in\",\"nulla\",\"id\",\"sunt\"],\"meta\":{\"link\":{\"self\":\"http://example.com/consectetur\",\"author\":\"http://example.com/minim\",\"collection\":\"http://example.com/sint\",\"replies\":\"http://example.com/amet\"}}},{\"id\":\"56fb50319138febd691cf0cc\",\"name\":\"Duncan Oneill\",\"isActive\":true,\"age\":33,\"registered\":\"2014-11-15T02:09:24\",\"tags\":[\"fugiat\",\"ea\",\"mollit\",\"culpa\",\"laborum\",\"reprehenderit\",\"sit\"],\"meta\":{\"link\":{\"self\":\"http://example.com/velit\",\"author\":\"http://example.com/cillum\",\"collection\":\"http://example.com/minim\",\"replies\":\"http://example.com/cupidatat\"}}},{\"id\":\"56fb503199c6c08c8afb0dad\",\"name\":\"Burke Nolan\",\"isActive\":true,\"age\":40,\"registered\":\"2014-06-04T01:04:40\",\"tags\":[\"ullamco\",\"ullamco\",\"in\",\"labore\",\"sit\",\"nulla\",\"Lorem\"],\"meta\":{\"link\":{\"self\":\"http://example.com/amet\",\"author\":\"http://example.com/sunt\",\"collection\":\"http://example.com/reprehenderit\",\"replies\":\"http://example.com/commodo\"}}},{\"id\":\"56fb503151f18f3ebb06b7e3\",\"name\":\"Bruce Keith\",\"isActive\":false,\"age\":25,\"registered\":\"2015-07-22T11:05:09\",\"tags\":[\"ea\",\"tempor\",\"laboris\",\"nostrud\",\"magna\",\"aute\",\"magna\"],\"meta\":{\"link\":{\"self\":\"http://example.com/non\",\"author\":\"http://example.com/eiusmod\",\"collection\":\"http://example.com/ad\",\"replies\":\"http://example.com/eiusmod\"}}}]"
```

Comment:
Not a big fan of this, as it adds a lot of bloat to the test file. You don't need JSON, you can just create an array of dictionaries in code.